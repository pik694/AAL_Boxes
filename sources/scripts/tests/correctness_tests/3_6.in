3
74 58 41
56 66 64
9 99 16

