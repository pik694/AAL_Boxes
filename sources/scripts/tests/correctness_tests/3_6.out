2 0 1 
426772->426772
